tests/cloudpickle_test.py

```diff
@@ -2352,6 +2371,71 @@ def test_lookup_module_and_qualname_stdlib_typevar():
     assert name == 'AnyStr'
 
 
+def test_lookup_module_and_qualname_external_module_remove():
+    import _cloudpickle_testpkg
+    T = _cloudpickle_testpkg.T
+
+    register_pickle_by_value(_cloudpickle_testpkg)
+    unregister_pickle_by_value(_cloudpickle_testpkg)
+    module_and_name = _lookup_module_and_qualname(T, name=T.__name__)
+    assert module_and_name is not None
+
+
+def test_lookup_module_and_qualname_external_module_remove_name():
```

Comment:
Is there any reason why these tests are not methods of the `CloudPickleTest` class?

Reply:
No explicit reason on my end, I simply put this test in the same location as the other lookup module and qualname tests, which are all outside that class. Doing the renaming now

Reply:
The reason for `test_lookup_module_and_qualname_dynamic_typevar` and other such tests to be outside of the `CloudPickleTest` class is that these tests do not include any explicit `cloudpickle.dumps` call: test sincluding such calls should be made methods of `CloudPickleTest` so that these calls are tested for  `protocol=HIGHEST_PROTOCOL` and `protocol=2`.

Reply:
Isn't that what we have now? `test_pickle_module_registered_for_pickling_by_value` invokes `cloudpickle.dumps` and `loads`, and exists within the `CloudPickleTest` class. `test_register_pickle_by_value_parents_and_children` and `test_register_pickle_by_value` do not utilise `dumps` or `loads`, and should be good to stay where they are?

Reply:
Not sure if it has always been the case (in that case, sorry about the un-necessary comments), but yes, the placement of tests look good now :)